.travis.yml

```diff
@@ -38,9 +38,11 @@ matrix:
   - python: 3.9
     env:
     - PLANETMINT_ACCEPTANCE_TEST=enable
+  - python: 3.9
     - PLANETMINT_INTEGRATION_TEST=enable
```

Comment:
I guess you missed the env: declaration . no tests/checks got executed. I assume that's because of the missing env: line

Reply:
oh yeah, should be fixed now! let's wait for the pipeline